src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
```

Comment:
The class `FigureSupplier` should not start with an empty line. Please remove any unnecessary empty lines at the beginning of class definitions.